.github/workflows/automated-dev-tests.yml

```diff
@@ -329,12 +330,20 @@ jobs:
           test-target: regression
       - name: Run HydroDyn tests
         uses: ./.github/actions/tests-module-hydrodyn
+        with:
```

Comment:
Just heads up that this module action doesn't have the option for unit vs regression test flag

Reply:
Ok.  I'll revert that change.

I don't think I understood exactly how that was setup, so I thought something was missing.

Reply:
Well, if the tests ran on GH Actions then its not exactly incorrect but it might be misleading to anyone reading it. I just wanted to point it out for your reference.

Reply:
thanks!